packages/help-center/src/components/notices.tsx

```diff
@@ -60,36 +60,39 @@ export const BlockedZendeskNotice: React.FC = () => {
 	);
 };
 
-export const EmailFallbackNotice: React.FC = () => {
+export const EmailFallbackNotice: React.FC< { isBlocked?: boolean } > = ( {
+	isBlocked = false,
+} ) => {
 	const navigate = useNavigate();
 	const { search } = useLocation();
 	const params = new URLSearchParams( search );
 	params.set( 'wapuuFlow', 'true' );
 	const url = '/contact-form?' + params.toString();
+
+	const title = isBlocked
+		? __( 'Live chat is currently unavailable.', __i18n_text_domain__ )
+		: __( 'Live chat is temporarily unavailable for scheduled maintenance.', __i18n_text_domain__ );
+
+	const message = __(
+		'Please reach out via <email>email</email> if you need immediate assistance.',
```

Comment:
I guess this is an oxymoron. 

```suggestion
		'Please reach out via <email>email</email> if you need assistance.',
```